src/components/ValidatedInput/ValidFieldPassword.tsx

```diff
@@ -18,24 +18,31 @@ import { Button } from "../Button";
 import { translation } from "../../utils/i18n";
 import "../../styles/icons/fa_icons.css";
 import DOMPurify from "dompurify";
+import zxcvbn from "zxcvbn";
+import { Enum } from "../../types";
 
 export interface ValidFieldPasswordProps extends Omit<ValidFieldTextProps, "type" | "rightIcon" | "isFloat"> {
   strengthLevelIndicator?: keyof typeof strengthLevelOrder;
```

Comment:
i think strengthLevelIndicator will no longer be used. please remove this.

Reply:
there still one component using this props on AP, I think we can remove it when we really do clean up and make sure we really not using it

Reply:
that one component is not used at all in AP, man. plus, it's only one, it should be easy to clean it up.

Reply:
I see, ok let me try to clean it up